controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1679,6 +1680,53 @@ bool master_thread::handle_cmdu_1905_topology_response(const std::string &src_ma
         }
     }
 
+    // The reported neighbors list might not be correct since the reporting al_mac hasn't received 
+    // a Topology Discovery from its neighbors yet. Therefore, remove a neighbor node only if more
+    // than 65 seconds have passed since we added this node. This promise that the reported al_mac
+    // will get the Topology Discovery messages from its neighbors and add them to the report.
+    auto last_state_change_timestamp = database.get_last_state_change(src_mac);
+    if (last_state_change_timestamp + std::chrono::seconds(65) < std::chrono::steady_clock::now()) {
+        LOG(TRACE) << "Checking if one of " << src_mac << " is no longer connected";
```

Comment:
I don't understand this comment - you never use src_mac after this line...

Reply:
Yes I was missing a word there. It should have been:
"Checking if one of " << src_mac << " **neighbors** is no longer connected"
fixed.